dask/diagnostics/profile_visualize.py

```diff
@@ -80,7 +92,7 @@ def get_colors(palette, funcs):
 
 
 def visualize(results, dsk, palette='GnBu', file_path=None,
-              show=True, **kwargs):
+              show=True, save=True, **kwargs):
```

Comment:
Would be nice if `budget` was checked for in `kwargs`. Could be unspecified, and just checked in a `kwargs.get('budget', 60)` if the signature feels a little long. Would need to be set in the calls to pprint_task as well.